src/pages/code-of-conduct.js

```diff
@@ -1,92 +1,92 @@
 import React from 'react';
-import Button from '../components/Button';
-import Card from '../components/Card';
-import CustomerCard from '../components/CustomerCard';
-import LabelText from '../components/LabelText';
 import Layout from '../components/layout/Layout';
-import SplitSection from '../components/SplitSection';
-import StatsBox from '../components/StatsBox';
-import customerData from '../data/customer-data';
-import HeroImage from '../svg/HeroImage';
-import SvgCharts from '../svg/SvgCharts';
 
 const Index = () => (
-    <Layout>
+  <Layout>
```

Comment:
The diff makes it look like I made massive changes, but it was mostly minor line formatting for consistency.